### PR TITLE
update docs for drone-ssh pluging.

### PR DESCRIPTION
**Pull request by appleboy**
as title

Signed-off-by: Bo-Yi Wu <appleboy.tw@gmail.com>

### CODE REVIEW

**Comment by appleboy** [comment]
@tboerger @bradrydzewski Can you help to review this doc?

File: content/appleboy/drone-ssh/index.md [hunk 1]
```diff
@@ -40,39 +40,42 @@ pipeline:
...
       - echo hello
       - echo world
++   proxy_host: 10.130.33.145
++   proxy_user: ubuntu
++   proxy_port: 22
++   proxy_password: 1234
 ```
 
-Example configuration for command timeout (unit: second), default value is 60 seconds:
+Example configuration for `master` branch:
```
Comment (tboerger) [inline content/appleboy/drone-ssh/index.md]: I would drop that, we are not documenting the conditions within the plugin index.

File: content/appleboy/drone-ssh/index.md [hunk 2]
```diff
@@ -82,66 +85,97 @@ pipeline:
...
-+   command_timeout: 10
     script:
       - echo hello
       - echo world
++   when:
++     branch: master
 ```
 
-Example configuration for execute commands on a remote server using ｀SSHProxyCommand｀:
+Example configuration for `tag` event:
```
Comment (tboerger) [inline content/appleboy/drone-ssh/index.md]: Same here.

File: content/appleboy/drone-ssh/index.md [hunk 3]
```diff
@@ -82,66 +85,97 @@ pipeline:
...
+  ssh:
+    image: appleboy/drone-ssh
+    host: foo.com
+    username: root
+    password: 1234
+    port: 22
++   secrets: [ aws_access_key_id ]
++   envs: [ aws_access_key_id ]
+    script:
+      - export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
```
Comment (tboerger) [inline content/appleboy/drone-ssh/index.md]: It is already en anv variable, so why this additional export?

File: content/appleboy/drone-ssh/index.md [hunk 4]
```diff
@@ -82,66 +85,97 @@ pipeline:
...
+ssh_username
+: account for target host user
+
+ssh_password
+: password for target host user
+
+ssh_key
+: plain text of user private key
+
+proxy_ssh_username
```
Comment (tboerger) [inline content/appleboy/drone-ssh/index.md]: Normally it should have the plugin name as an index?

File: content/appleboy/drone-ssh/index.md (hunk 1 above)
Comment (appleboy) [reply content/appleboy/drone-ssh/index.md]: Done.

File: content/appleboy/drone-ssh/index.md (hunk 2 above)
Comment (appleboy) [reply content/appleboy/drone-ssh/index.md]: Done.

File: content/appleboy/drone-ssh/index.md (hunk 3 above)
Comment (appleboy) [reply content/appleboy/drone-ssh/index.md]: See: 

Issue: https://github.com/appleboy/drone-ssh/issues/89
PR: https://github.com/appleboy/drone-ssh/pull/91

File: content/appleboy/drone-ssh/index.md (hunk 4 above)
Comment (appleboy) [reply content/appleboy/drone-ssh/index.md]: What do you mean?

Comment (tboerger) [reply content/appleboy/drone-ssh/index.md]: Normally the variables got the plugin name as a prefix, e.g. ssh_proxy_username

Comment (appleboy) [reply content/appleboy/drone-ssh/index.md]: @tboerger OK I will update that.

**Comment by appleboy** [comment]
@tboerger Please help to merge this PR.

**Pull request merged**